recipes/ipydagred3/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "ipydagred3" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a1acc34f25909a1fa5d5bc89c8fcf7874a968fcefb71a2f80cb2c0ef22eca6af
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
```

```suggestion
  script: "{{ PYTHON }} -m pip install . -vv --no-build-isolation"
```
you shouldn't need the above tag, but maybe